src/components/Todo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+
+function Todo({ todo: { user, title, completed } }) {
+  return (
+    <>
+      <User user={user} />
```

Comment:
три слова user подряд не прибавляют коду читабельности, давай компоненту и его пропсам разные имена, или делай так 
```suggestion
      <User {...user} />
```